_search-plugins/sql/settings.md

```diff
@@ -78,3 +78,39 @@ Setting | Default | Description
 `plugins.sql.cursor.keep_alive` | 1 minute | Configures how long the cursor context is kept open. Cursor contexts are resource-intensive, so we recommend a low value.
 `plugins.query.memory_limit` | 85% | Configures the heap memory usage limit for the circuit breaker of the query engine.
 `plugins.query.size_limit` | 200 | Sets the default size of index that the query engine fetches from OpenSearch.
+
+## Spark connector settings
+
+The SQL plugin supports [Apache Spark](https://spark.apache.org/) as an augmented compute source. When data sources are defined as tables in Apache Spark, OpenSearch can consume those tables. This allows you to run SQL queries against external sources inside OpenSearch Dashboard's [Discover]({{site.url}}{{site.baseurl}}/dashboards/discover/index-discover/) and observability logs. 
+
+To get started, enable the following settings to add Spark as a data source and enable the correct permissions.
+
+Setting | Description
+:--- | :---
+`spark.uri` | The identifier for your Spark data source.
+`spark.auth.type` | The authorization type used to authenticate into Spark.
+`spark.auth.username` | The username for your Spark data source.
+`spark.auth.password` | The password for your Spark data source.
+`spark.datasource.flint.host` | The host of the Spark data source. Default is `localhost`.
+`spark.datasource.flint.port` | The port number for Spark. Default is `9200`.
+`spark.datasource.flint.scheme` | The data scheme used in your Spark queries. Valid values are `http` and `https`.
+`spark.datasource.flint.auth` | The authorization required to access the Spark data source. Valid values are `false` and `sigv4`.
+`spark.datasource.flint.region` | The region in which your OpenSearch cluster is located. Only use when `auth` is set to `sigv4`. Default value is `us-west-2``.
```

Comment:
When referring to an AWS Region, always use "AWS Region" on first appearance. Capitalized "Region" may be used for subsequent appearances.